testsuite/tests/libratest/smoke_test.rs

```diff
@@ -421,3 +471,28 @@ fn test_external_transaction_signer() {
         _ => panic!("Signed transaction payload expected to be of struct Script"),
     }
 }
+
+#[test]
+fn test_full_node_basic_flow() {
+    // launch environment of 4 validator nodes and 2 full nodes
+    let mut env = TestEnvironment::new(4);
+    env.setup_full_node_swarm(2);
+    env.launch_swarm(RoleType::Validator);
+    env.launch_swarm(RoleType::FullNode);
+
+    // execute smoke script
+    let validator_ac_client = env.get_validator_ac_client(0);
+    test_smoke_script(validator_ac_client);
+
+    // read state from full node client
+    let mut full_node_client = env.get_full_node_ac_client(0);
+    full_node_client.create_next_account(false).unwrap();
+    assert_eq!(
+        Decimal::from_f64(7.0),
```

Comment:
7 looks like a magic number here. I see that `test_smoke_script` makes some txns which results in balance of a/c 0 being 7.0. Could we have the function return the final state of each a/c it creates?

Other changes lgtm